### DISCUSSION TITLE
.NET MAUI

### DISCUSSION THREAD

**Discussion by Tanzy** [shinyorg/shiny#1157] [Q&A] [answered]
We are currently looking at updating a Xamarin Forms application to MAUI and wanted to start using this framework

Is there any idea as to when an RC of the MAUI version will become available?

Cheers

**Comment by aritchie** [comment, marked as answer]
As with all OSS, when it is done.  I'm the only maintainer, so I work on it as I feel like it and as it serves my customer base.

**Reply by Tanzy** [reply]
Appreciate that, just know that sometimes people have an idea of what they are looking at timescale-wise. Figured it was worth asking the question

**Comment by Tanzy** [comment]
Is there anywhere a list of outstanding things that need to be done? The website says `This is an ALPHA release! There are still changes planned!` but can't see any indication as to what they are

**Reply by aritchie** [reply]
There isn't a public list.  I'm currently evaluating APIs for change or refactor.
